server/pse/pse_zos.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build zos
+// +build zos
+
+package pse
+
+// This is a placeholder for now.
+func ProcUsage(pcpu *float64, rss, vss *int64) error {
```

Comment:
Possible to make real?

Reply:
[Go landed on zOS only a couple of years ago](https://community.ibm.com/community/user/ibmz-and-linuxone/blogs/james-tang1/2021/03/12/ibm-open-enterprise-sdk-for-go-is-now-available), and platform-specific bits are still being upstreamed.
Besides, _my_ experience with Go is quite (very) limited; so I'm taking it one step at a time :)